timeline/templates/timeline/_list_item.html

```diff
@@ -31,6 +31,19 @@
     {% endif %}
   </td>
   <td>
+    {% check_superuser user as perm %}
+    {% if event.project and perm and site_admin %}
+        <span class="badge"
+              title="{{ event.project }}"
+              data-toggle="tooltip" data-placement="top">
+        <i class="iconify"
+             data-icon="{% if event.project.type == 'PROJECT' %}mdi:cube{% else %}mdi:rhombus-split{% endif %}">
+        </i>
+        <a href="{% url 'projectroles:detail' project=event.project.sodar_uuid %}">
```

Comment:
Please format templates with a 2-space indent. As an exception to this, when splitting a long tag on multiple lines, it's ok to place the attributes starting in the same position for clarity. E.g. above here in the icon, "class" and "data-icon" would be aligned on the same position.

Reply:
I definetely should learn more about html formating! Thanks, fixed